yt/visualization/volume_rendering/old_camera.py

```diff
@@ -1193,15 +1191,15 @@ class InteractiveCamera(Camera):
     frames = []
 
     def snapshot(self, fn=None, clip_ratio=None):
-        from matplotlib import pylab as pylab
+        import matplolib.pyplot as plt
 
-        pylab.figure(2)
+        plt.figure(2)
```

Comment:
Any reason not to use `self._pyplot` here, or to not use `plt` elsewhere? I am confused by the code tbh.

Reply:
I merely replaced the existing code. I don't think it was designed or refined really. It looks to me no one touched it after it first worked. I'll refactor so that it doesn't confuse anyone else later.

Reply:
should be better now